src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchViewModel.cs

```diff
@@ -294,6 +399,21 @@ public void Sort()
             this.SearchResults = temp;
         }
 
+        /// <summary>
+        /// Based on the known hosts received from back end,
```

Comment:
maybe specify the backend is the package manager server?

Reply:
Will do